### ISSUE TITLE
Parameters of nn.Conv2d in MetaAconC

### ISSUE THREAD

**Issue by dosemeion** [nmaac/acon#1]
MetaAconC:
self.fc1 = nn.Conv2d(width, max(r,width//r), kernel_size=1, stride=1, bias=True)
self.bn1 = nn.BatchNorm2d(max(r,width//r))
self.fc2 = nn.Conv2d(max(r,width//r), width, kernel_size=1, stride=1, bias=True)
self.bn2 = nn.BatchNorm2d(width)

It should be “nn.Conv2d(width, max(r,width//r), kernel_size=1, stride=1, bias=False)”？

**Comment by nmaac** [comment]
Hi @dosemeion , we set bias=True here because initially we used another closed-source framework which has bias=True. Therefore the trained models have bias=True. This does not affect the accuracy.
